### PR TITLE
BUG: Fixing of the sys library issue in Slicer 4.4

### PR DESCRIPTION
**Pull request by jbvimort**

### CODE REVIEW

**Comment by fbudin69500** [comment]
Can you remove the debugging "print"? Thanks.

**Comment by jbvimort** [comment]
Done!

File: AnglePlanes/AnglePlanes.py [hunk 1]
```diff
@@ -503,6 +506,8 @@ def onComputeBox(self):
             dim.append(bound[x * 2 + 1] - bound[x * 2])
             origin.append(bound[x * 2] + dim[x] / 2)
             dim[x] *= 1.1
+        print dim
+        print origin
```
Comment (fbudin69500) [inline AnglePlanes/AnglePlanes.py]: Can you remove these too?

**Comment by jbvimort** [comment]
Done to!

**Pull request merged**